docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Code Builder Quick Start
+lang: en
+---
+
+## Overview
+Complete this Quick Start to become familiar with the Code Builder interface and perform some simple tasks in your Code Builder environment.
+## Important Considerations for Code Builder Beta
+We've capped usage for beta at 20 hours for a maximum of 30 days. We highly recommend that you save your work and close the browser tab that is running Code Builder to stop the usage clock when you aren’t using it.
+
+### Don’t Forget to Save Your Work
+Working in the cloud has its advantages. However, unlike working on a desktop where you save your files to a local machine, you must either deploy your changes to your org, or commit them to source control to save your work. Save your work before you close the Code Builder tab on your browser so you don’t lose it. 
+
+**Note:** Throughout the beta, Code Builder environments could be deleted.  All beta environments will be removed before GA.
+
+
+## Let’s Get Started
+To get started you’ll do the following:
+1. Create a Code Builder environment.
+2. Create a Salesforce DX project.
+3. Connect to your org by logging in.
+Now you’re ready to use Code Builder.
+
+## Connect to Your Dev Environment
+1. From the App Launcher, find and open **Code Builder**.
+
+**Note:** If you don't see Code Builder as an option, or you see an empty screen when you launch it, contact your admin to make sure you have the correct license and permission set assigned to you.
+
+2. Click **Get Started**.
+3. Click **New Project** to create a new Salesforce DX project.
+4. Give your project a name and choose **Standard** for project type.
+
+Next, connect an org to your Code Builder environment. You can choose to connect to a Salesforce org or a sandbox org.
+
+1. Log in to your development environment, click **Allow** to grant access, then click **Next**.
+2. Choose an alias (nickname) for your org. If you work on many orgs, make sure the nickname helps you identify the org quickly in the future. 
+Your new Code Builder environment is now available on your dashboard.
+3. Click **Launch** to launch Code Builder in a new tab. 
+4. Now sit back and relax for a few minutes while Code Builder creates and configures your workspace. Code Builder might take a fews minutes to start up when it initially sets up.
+
+**Tip**: Once you have created your Code Builder environment, launch it from the dashboard at any time, or bookmark it for fast access.
+
+## Let's Take A Quick Tour
+You’re now in your developer environment in Code Builder. Code Builder has VS Code’s rich IDE plus it gives you easy access to cool Salesforce development-specific tools through the Salesforce extension pack. Let’s take a quick tour and get to know the lay of the land. Your screen looks something like this:
```

Comment:
I was told many years ago that using a possessive in a product name can nullify the trademark. So just thinking about VS Code here and how you could rewrite to avoid the possessive.

Reply:
Although with the rich IDE provided by VS Code, Code Builder gives you easy access to...